server/src/controller/apis/blogs.js

```diff
@@ -1,18 +1,21 @@
 import { Router } from 'express';
 import authInstance from '../../configs/authentication';
-import { authenticateUserBlog, blogExists } from '../../middlewares/blogs';
+import { authenticateUserBlog, createBlogValid } from '../../middlewares/blogs';
 import blogsAPI from '../../services/blogs/api';
+import {
+  saveBlog, fetchBlog, updateBlog, deleteBlog,
+} from '../../services/blogs/service';
```

Comment:
I would suggest not using the service here. Use only the middlewares only. Services don't make a sense here.